targets/HUDL/write_to_display.cpp

```diff
@@ -63,7 +63,7 @@ int main() {
         // Clear LCD Screen
         // clear_lcd();
 
-        board.drivePixel(1, 1, 1, 255);
+        board.driveColumn(1, 1, 1, 255);
```

Comment:
Does this example still make sense given what `initLCD()` does? Is it worth keeping this example target here if there's already a bunch of examples in EVT-core?